spec/user_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
🚀 Well done on writing sufficient and passing unit tests for all model validations 💯 

![image](https://github.com/MarwanDev/blog-app/assets/38088818/c61e8791-e17b-4bcb-b33e-f8e31db44001)